src/TodoList/index.tsx

```
@@ -0,0 +1,77 @@
+          onChange={onChangeInputText}
+          onSubmit={textInputHandler}
+          inputText={inputText}
+        />
+        {todoList.map((item) => (
+          <TodoItem
+            id={item.id}
+            text={item.text}
+            completed={item.completed}
+            onClickDelete={textDeleteHandler}
+            onClickUpdate={textUpdateHandler}
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)